syntax = "proto3";

option java_multiple_files = true;
package ffvms;

service Greeter {
    rpc SayHello(HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
    string name = 1;
}

message HelloResponse {
    string message = 1;
}